server/src/main/java/org/opensearch/extensions/ExtensionRequest.java

```diff
@@ -26,54 +26,54 @@
  */
 public class ExtensionRequest extends TransportRequest {
     private static final Logger logger = LogManager.getLogger(ExtensionRequest.class);
-    private final ExtensionsManager.RequestType requestType;
-    private final Optional<String> uniqueId;
+    private final ExtensionRequestProto.ExtensionRequest request;
```

Comment:
Can we import `ExtensionRequest` like `import org.opensearch.extensions.proto.ExtensionRequestProto.ExtensionRequest`?

Reply:
Unfortunately this class is also ExtensionRequest which conflicts. 
We can change the names if you strongly believe. Im ok either way.

Reply:
Ah. I see